Dockerfile

```diff
@@ -26,9 +26,11 @@ COPY --from=builder /jellyfin /jellyfin
 COPY --from=ffmpeg /ffmpeg-bin/* /usr/bin/
 EXPOSE 8096
 VOLUME /config /media
+
+# libfontconfig1 is required for Skia
 RUN apt-get update \
  && apt-get install --no-install-recommends --no-install-suggests -y \
-   libfontconfig1 # Required for Skia \
+   libfontconfig1 \
```

Comment:
Would this not work with the slash *before* the comment?

```
 && apt-get install --no-install-recommends --no-install-suggests -y \
   libfontconfig1 \ # Required for Skia
   libblah1 \ # Required for blah
```

Though honestly I'd prefer the following as it's far easier to read long-term:

```
 && apt-get install --no-install-recommends --no-install-suggests -y \
    # Required for Skia
   libfontconfig1 \
    # Required for blah
    libblah1 \
    # etc.
```

Reply:
No. I tried that in an unreleased test from the PR i put up before #606 that touched on this. It breaks it entirely... Because of course it does.

Reply:
Does the second method do the same? Damn you Docker...

Reply:
Yup.

Reply:
You usually aren't expected to comment your dockerfil. The file itself is your documentation so to speak. Sure that's why it has such limited support for commenting and adding documentation to the file itself.

Reply:
I mean we're limited then, but holy crap is that a terrible attitude from their side... More reasons I dislike Docker.

Reply:
That's news to me, ugh.